.gitignore

```diff
@@ -4,6 +4,13 @@
 # ignore cmake build directory
 /build
 
+# ignore generated documentation directories
+/html
+/latex
+
+# ignore docker artifact directory
+/tools/docker/deps
+
```

Comment:
This is related to the Dockerfile.tensorrt (see below), which I used for local testing. I can remove this if that docker file is not supposed to be part of this PR?